### PR TITLE
Chapters reorganization

### PR DESCRIPTION
**Pull request by Honzan9**
Hi @0ptim, I made first proposal of grouping the chapters (pages) to more logical scheme, but it is just from my point of view. So let me know what do you think about it. The recently discussed split of Invest & Trade page is included and I think, that it was a good idea ;-).   Looking forward to your feedback!

### CODE REVIEW

**Comment by 0ptim** [comment]
Hi @Honzan9 
I very much like it. Way better than before.

I think `Investing_Trading.md` is no longer needed, right?
Attention: The page is linked from other pages:
image

**Comment by Honzan9** [comment]
Hi @0ptim, I'm glad you like it. I think there remains some work before merge. At least make only one page for DEX. I wana go through each group now and if it will be possible I wana add/remove pages, that I think they should be or not in current state of content.

**Comment by 0ptim** [comment]
Great! So, will mark your Pull Request as `draft`.

You can create PRs as `draft` to let other people look at what you did and get feedback, but clearly communicate that it is not yet ready to be merged.
=> https://www.youtube.com/watch?v=l0Lw_zPX4TA

**Comment by Honzan9** [comment]
Ok, I will do it next time like a draft ;-)

**Comment by Honzan9** [comment]
> Hi @Honzan9 I very much like it. Way better than before.
> 
> I think `Investing_Trading.md` is no longer needed, right? Attention: The page is linked from other pages: image

Yes, as we agreed that the split version of I&T page is better, it is no longer needed. I'll remove it in the next commit and also will adjust the links.

**Comment by Honzan9** [comment]
Hey @0ptim, I went through the content and there is a lot of work yet :-D I think that the right way now is to update the pages one by one in each of main groups/chapters. I've already started on the DEX content (new PR).
So the current state of this PR (after my last commit) is IMO ready to review and merge.

**Comment by 0ptim** [comment]
Hey @Honzan9 
Great! I'll take a look as soon as I get time. In the meantime, I'll convert this draft PR to a normal PR. For the next time you can do this too to signal me, that the PR is ready to be reviewed and merged. 😊

image

File: package.json
Comment (0ptim) [inline package.json:1]: Hey @Honzan9
Did you push this on purpose? Maybe this should not be on "de" by default. If you want to launch in "de" locally, you can adjust it, but should not commit/push it to the remote 😊

**Comment by Honzan9** [comment]
Hi @0ptim, "package.json" corrected.

**Pull request merged**